packages/clerk-js/README.md

```diff
@@ -49,8 +49,8 @@ Once you have installed the package, you will need to import the ClerkJS object
 ```js
 import Clerk from '@clerk/clerk-js';
 
-const clerkFrontendApi = 'pk_[publishable_key]';
-const clerk = new Clerk(clerkFrontendApi);
+const clerkPublishableKey = 'pk_[publishable_key]';
```

Comment:
You'll need to update the text in line 47, too

(And check if the rest of the README still references Frontend API)